src/CONST.ts

```diff
@@ -2737,6 +2737,8 @@ const CONST = {
     },
 
     MISSING_TRANSLATION: 'MISSING TRANSLATION',
+    SEARCH_MAX_LENGTH: 500,
+    ERROR_EXCEED_RANGE: 20,
```

Comment:
```suggestion
    /**
      * The count of characters we'll allow the user to type after reaching SEARCH_MAX_LENGTH in an input.
      */
    ADDITIONAL_ALLOWED_CHARACTERS: 20,
```

Thanks! Now that I understand what this is, I can suggest what I think is a better name along with some nice docs.

Reply:
Yes, the comment makes it more understandable.

Reply:
What name do you suggest?

Reply:
Yes, correct, I missed that.

Reply:
@flodnv updated.

Reply:
Thanks!